### DISCUSSION TITLE
Add Nested Container Support for Elementor Slider Widget

### DISCUSSION THREAD

**Discussion by ayoubkhan558** [elementor/elementor#22715] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Currently, it is not possible to add Elementor widgets within the Slider widget. This limitation restricts the flexibility and creative possibilities for designing dynamic and interactive sliders.

### Describe the solution you'd like

I would like to request the implementation of a feature that enables nested container support within the Elementor Slider widget. This would allow users to incorporate any Elementor widget as a slide content, similar to the way tabs can be created with nested containers.

### Describe alternatives you've considered

As a workaround, users might currently resort to custom code or third-party plugins to achieve the desired functionality. However, having native support for nested containers within the Slider widget would provide a more intuitive and streamlined experience for Elementor users.

### Additional context

The ability to use Elementor widgets within the Slider widget would empower designers and developers to create more engaging and dynamic sliders, improving the overall user experience of websites built with Elementor. This enhancement would greatly expand the creative possibilities and boost the productivity of Elementor users.

**Comment by ghost** [comment]
+1

**Comment by ayoubkhan558** [comment]
### Looks like already implemented +1

![image](https://github.com/elementor/elementor/assets/99708427/cc7318cf-d9c8-4817-8da4-d599da8c0422)
